File: data/test_dataset/Python/palindrome.py
def palindrome(num):
    temp = num
    rev = 0
    while(num > 0):
        dig = num % 10
        rev = rev * 10 + dig
        num = num // 10
    if(temp == rev):
        print("Number is palindrome")
    else: 
        print("Number is not palidrome")

palindrome(131)

File: data/test_dataset/R/palindrome.r
palindrome <- function(num){
    rev = 0
    temp = num
    while (num > 0) {
      dig = num %% 10
      rev = rev * 10 + dig
      num = num %/% 10
    }
    if (rev == temp)
    {
     return (TRUE)
    }
    else{
      return (FALSE)
    }
}
